src/App.tsx

```diff
@@ -1,61 +1,123 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+
+function getUser(userId: number | null): User | null {
+  const foundUser = usersFromServer.find(user => user.id === userId);
+
+  return foundUser || null;
+}
+
+export const todos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUser(todo.userId),
+}));
+
+const maxTodosId = [...todos].sort((todo1, todo2) => todo2.id - todo1.id)[0].id;
```

Comment:
this value still calculates just one time. And each new todo will be with id bigger than 1 that its value (so with the same id)

all new todos with key 16:
<img width="459" alt="Screenshot 2022-09-26 at 21 56 07" src="https://user-images.githubusercontent.com/47606379/192357642-05d5e0bc-56e8-4e1d-9268-b274b4039ae6.png">